elixir/apps/domain/lib/domain/events/hooks/policies.ex

```diff
@@ -69,7 +69,9 @@ defmodule Domain.Events.Hooks.Policies do
       payload = {:reject_access, policy_id, actor_group_id, resource_id}
       :ok = PubSub.ActorGroup.Policies.broadcast(actor_group_id, payload)
 
-      Flows.expire_flows_for_policy_id(account_id, policy_id)
+      # TODO: WAL
+      # Broadcast flow side effects directly
+      :ok = Flows.expire_flows_for_policy_id(account_id, policy_id)
```

Comment:
Just curious, did you add the match on `:ok` to make sure that if anything else is returned other than `:ok` that we fail hard?

Reply:
Yeah, needed to rebase

Reply:
Wait, no this has been changed so that we _don't_ short circuit the remaining broadcasts in case a single one fails. We want to optimistically try to send all broadcasts.